### PR TITLE
Replace tripleo-repos by repo-setup

### PR DESCRIPTION
**Pull request by kajinamit**
... because TripleO is being deprecated.

### CODE REVIEW

**Review by fao89 — APPROVED** [review]

**Review by raukadah — APPROVED** [review]

**Comment by Sandeepyadav93** [comment]
/lgtm cancel

~~~
$ repo-setup current-podified-dev -o /tmp/temporary
WARNING: Unsupported platform 'fedora33' detected by repo-setup, centos9 will be used unless you use CLI param to change it.
Installed repo delorean-master-testing to /tmp/temporary/delorean-master-testing.repo
Installed repo delorean-current-podified to /tmp/temporary/delorean-current-podified.repo
Installed repo delorean to /tmp/temporary/delorean.repo
Installed repo repo-setup-centos-highavailability to /tmp/temporary/repo-setup-centos-highavailability.repo
Installed repo repo-setup-centos-powertools to /tmp/temporary/repo-setup-centos-powertools.repo
Installed repo repo-setup-centos-appstream to /tmp/temporary/repo-setup-centos-appstream.repo
Installed repo repo-setup-centos-baseos to /tmp/temporary/repo-setup-centos-baseos.repo
Cache was expired
0 files removed
~~~

Seeing two files, delorean-current-podified.repo and delorean.repo , checking if it is expected or not.

**Comment by Sandeepyadav93** [comment]
/lgtm
/approve

Discussed with chandan, repo-setup will need a fix for duplicate delorean.repo, but we are already utilizing the repo-setup here: https://github.com/openstack-k8s-operators/install_yamls/blob/main/devsetup/edpm/services/dataplane_v1beta1_openstackdataplaneservice_reposetup.yaml#L26

lets merge this and we will fix repo-setup in a follow-up.

**Comment by Sandeepyadav93** [comment]
/test images

Mirror issue last time, checking if its consistent.

File: images/tempest-container/Dockerfile [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 USER root
 RUN dnf install -y gcc git jq make python3 python3-pip python-setuptools python-requests
-RUN git clone https://git.openstack.org/openstack/tripleo-repos && \
-    pushd tripleo-repos && \
+RUN git clone https://github.com/openstack-k8s-operators/repo-setup && \
+    pushd repo-setup && \
     python3 setup.py install && \
     popd && \
-    /usr/local/bin/tripleo-repos current-tripleo-dev
+    /usr/local/bin/repo-setup current-podified-dev
```
Comment (viroel) [inline images/tempest-container/Dockerfile:8]: some packages are missing from master components, can we move to current-podified antelope like was recently made on install_yamls[1]?

[1] https://github.com/openstack-k8s-operators/install_yamls/blob/8c040b22b2c07ea2a08cdbc9eac73bfed16f6038/devsetup/edpm/services/dataplane_v1beta1_openstackdataplaneservice_reposetup.yaml#L26

Comment (kajinamit) [reply images/tempest-container/Dockerfile]: I've updated this accordignly.

**Pull request merged**